docs/books/admin_guide/12-network.md

```diff
@@ -119,27 +119,80 @@ There are also specific addresses within a network, which must be identified. Th
+    * Software-level spoofing (temporary): Modifies how the MAC address appears to the operating system. These changes are usually reset after a system reboot. The MAC address of the virtual network card in the virtual host is also implemented through spoofing.
+
+An Internet Protocol (**IP**) address is an identification number permanently or temporarily assigned to each device connected to a computer network using the Internet Protocol. The IP address and subnet mask must appear in pairs, which is determined by the basic principles of network communication. Through the subnet mask, we can know the current IP address:
+
+* network bits and host bits
+* NetID or SubnetID
+* HostID
+* network address
+* broadcast address
+
+IP addresses are classified based on the version field in the packet as follows:
+
+* **IPv4‌** - (4 bit, 0100). The available quantity of IPv4 is 2<sup>32</sup> (known from the source and destination address fields in IPv4 packets). Specifically divided into:
+
+    * Class A address. Its range is from **0.0.0.0** to **127.255.255.255**
+    * Class B address. Its range is from **128.0.0.0** to **191.255.255.255**
+    * Class C address. Its range is from **192.0.0.0** to **223.255.255.255**
+    * Class D address. Its range is from **224.0.0.0** to **239.255.255.255**
+    * Class E address. Its range is from **240.0.0.0** to **255.255.255.255**
+
+    Among them, Class A addresses, Class B addresses, and Class C addresses all have their own private address ranges. 0.0.0.0 is a reserved address and is not assigned to the host. Class D addresses are used for multicast communication and are not assigned to hosts. Class E addresses are reserved and not used for regular networks.
+
+    Due to space limitations, we will not provide a detailed explanation of the content of IPv4 packets here.
+
+* **IPv6** - (4 bit, 0110). The available quantity of IPv6 is 2<sup>128</sup> (known from the source and destination address fields in IPv6 packets). Specifically divided into:
 
-An Internet Protocol (**IP**) address is an identification number permanently or temporarily assigned to each device connected to a computer network using the Internet Protocol.
-One part defines the network address (NetID or SubnetID as the case may be), the other part defines the address of the host in the network (HostID). The relative size of each part varies according to the network (sub)mask.
+    * Unicast address. Include Link-local unicast address (LLA), Unique local address (ULA), Global unicast address (GUA), Loopback address, Unspecified address
+    * Anycast address
+    * Multicast address
 
-An IPv4 address defines an address on 4 bytes. The number of available addresses being close to saturation a new standard was created, the IPv6 defined on 16 bytes.
+    Due to space limitations, we will not provide a detailed explanation of the content of IPv6 packets here.
```

Comment:
There is no space limitation. Lol.

Please remove this sentence and simply leave out detailed explanation of the content of IPv6 packets.  i.e. no need to draw extra attention to the missing info !!